ewwep.py

```diff
@@ -658,14 +658,15 @@ def __init__(
 		user_isgeneral = user_data.life_state == ewcfg.life_state_kingpin
 		user_isjuvenile = user_data.life_state == ewcfg.life_state_juvenile
 		user_isdead = user_data.life_state == ewcfg.life_state_corpse
+		user_isexecutive = user_data.life_state == (ewcfg.life_state_executive or ewcfg.life_state_lucky)
```

Comment:
this won't work

Reply:
added an is_executive and is_lucky operator, should work correctly now